_data/transport.yml

```diff
@@ -17,6 +17,12 @@ websites:
       img: ba.png
       tfa: No
 
+    - name: EasyJet
```

Comment:
easyJet seem to use a lowercase `e` on all platforms. Perhaps you could change the name to meet their standards?